src/main/java/duke/Parser.java

```diff
@@ -36,6 +27,8 @@ public static Command parse(String input) throws DukeException {
             return new MarkCommand(Integer.parseInt(remaining));
         } else if (input.matches("unmark [1-9]+\\d*")) {
             return new UnmarkCommand(Integer.parseInt(remaining));
+        } else if (firstWord.equals("find")) {
+            return new FindCommand(remaining);
         } else if (firstWord.equals("todo")) {
```

Comment:
Might want to consider using a switch statement instead of an if-else statement. The organised nature of switch-statements make collaboration between teammates easier. It looks neater as well :).